sdk/core/Azure.Core.Experimental/src/Variant/Variant.cs

```diff
@@ -994,67 +994,67 @@ public static Variant Create<T>(T value)
         {
             // Explicit cast for types we don't box
             if (typeof(T) == typeof(bool))
-                return new(Unsafe.As<T, bool>(ref Unsafe.AsRef(value)));
+                return new(Unsafe.As<T, bool>(ref Unsafe.AsRef(in value)));
```

Comment:
See: Warning [CS9192: Argument should be passed with ref or in keyword.](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/ref-modifiers-errors?f1url=%3FappId%3Droslyn%26k%3Dk(CS9191)#reference-variables-require-a-referent)